src/index.ts

```diff
@@ -0,0 +1,353 @@
+
+        const { accountNumber, networkId, exchangeAddress } =
+          getStandardParams(argv);
+
+        const wallet = loadAccount(networkId, accountNumber);
+        const exchange = IExchange__factory.connect(exchangeAddress, wallet);
+
+        try {
+          const trade = await exchange.callStatic.changePosition(
+            deltaAsset,
+            deltaStable,
+            stableBound
+          );
+
+          console.log({
+            startAsset: trade.startAsset.toString(),
+            startStable: trade.startStable.toString(),
+            totalAsset: trade.totalAsset.toString(),
+            totalStable: trade.totalStable.toString(),
+            tradeFee: trade.tradeFee.toString(),
+            traderPayout: trade.traderPayout.toString(),
+          });
+        } catch (e) {
+          console.log("Can not estimate trade");
+          console.log({ e });
+        }
+      }
+    )
+    .command(
+      ["approveTokens"],
+      "approve_tokens",
+      async (yargs: Argv) => yargs,
+      async (argv: any) => {
+        const { accountNumber, networkId, exchangeAddress } =
+          getStandardParams(argv);
+
+        const wallet = loadAccount(networkId, accountNumber);
+        const exchange = IExchange__factory.connect(exchangeAddress, wallet);
+
+        const assetTokenAddress = await exchange.assetToken();
+
+        const assetToken = IERC20__factory.connect(assetTokenAddress, wallet);
+
+        const tx1 = await assetToken.approve(
+          exchangeAddress,
+          ethers.utils.parseEther("100000")
+        );
+        await tx1.wait();
+
+        const stableTokenAddress = await exchange.stableToken();
+
+        const stableToken = IERC20__factory.connect(stableTokenAddress, wallet);
+
+        const tx2 = await stableToken.approve(
+          exchangeAddress,
+          ethers.utils.parseEther("100000")
+        );
+        await tx2.wait();
+
+        console.log(
+          "Approved both tokens for account: " + (await wallet.getAddress())
+        );
+      }
+    )
+    .command(
+      ["liquidate"],
+      "liquidate",
+      async (yargs: Argv) => {
+        return yargs.option("trader", {
+          alias: "t",
+          describe: "trader",
+          type: "string",
+          require: true,
+        });
+      },
+      async (argv: any) => {
+        const { accountNumber, networkId, exchangeAddress } =
+          getStandardParams(argv);
+
+        const wallet = loadAccount(networkId, accountNumber);
+
+        const exchange = IExchange__factory.connect(exchangeAddress, wallet);
+
+        const tx = await exchange.liquidate(argv.trader);
+
+        const receipt = await tx.wait();
+
+        console.log("Liquidated in tx: " + receipt.transactionHash);
+      }
+    )
+    .command(
+      ["estimateLiquidate"],
+      "estimate_liquidate",
+      async (yargs: Argv) => {
+        return yargs.option("trader", {
+          alias: "t",
+          describe: "trader",
```

Comment:
We could probably say `describe: "trader address"`.
Or be more verbose and explain it in full:
```javascript
    describe: "Address of a trader that we are checking for liquidation.\n\
Note that this command is not running any transactions, so the estimation is free."
```